app/src/main/res/layout/row_item_description.xml

```diff
@@ -1,52 +1,89 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:layout_marginTop="12dp"
+  android:layout_marginBottom="@dimen/activity_margin_horizontal"
+  android:layout_marginStart="@dimen/activity_margin_horizontal"
+  android:layout_marginEnd="@dimen/activity_margin_horizontal"
+  android:layout_marginLeft="@dimen/activity_margin_horizontal"
+  android:layout_marginRight="@dimen/activity_margin_horizontal"
+  android:elevation="6dp">
+
+  <LinearLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:orientation="horizontal"
-    android:weightSum="8">
+    android:orientation="vertical">
 
-    <androidx.appcompat.widget.AppCompatSpinner
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="10dp"
+      android:layout_marginBottom="10dp"
+      android:layout_marginStart="20dp"
+      android:layout_marginEnd="20dp">
+
+      <ImageView
+        android:id="@+id/btn_remove"
+        android:layout_width="24dp"
+        android:layout_height="wrap_content"
+        android:layout_margin="5dp"
+        android:layout_alignParentEnd="true"
+        android:contentDescription="@string/remove"
+        android:visibility="visible"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/ic_clear_24dp" />
+
+      <Spinner
         android:id="@+id/spinner_description_languages"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_weight="0"
-        android:minWidth="@dimen/tiny_height"
-        android:padding="@dimen/dimen_0"
+        android:layout_alignParentStart="true"
```

Comment:
this attribute is useless in a constraintlayout I'd hazard

Reply:
Forgot to remove it, thanks